cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1001,12 +1001,8 @@ function Configure_Dockerd_CniNetworking {
   $cidr_range_start = Get_PodIP_Range_Start(${env:POD_CIDR})
 
   # Explanation of the CNI config values:
-  #   CLUSTER_CIDR: the cluster CIDR from which pod CIDRs are allocated.
   #   POD_CIDR: the pod CIDR assigned to this node.
   #   CIDR_RANGE_START: start of the pod CIDR range.
-  #   MGMT_SUBNET: the subnet on which the Windows pods + kubelet will
```

Comment:
Can you clean up the code for `mgmt_subnet` throughout the file? It looks like most/all of it is now unused.

Reply:
removed.

Reply:
cleaned up